src/app/static/src/app/components/Warning.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+    <div v-if="warnings.length > 0">
```

Comment:
This isn't strictly necessary as the warning alert component filters out empty warnings. Just added for robustness and so that the component can stand alone theoretically.